common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Settings.scala

```diff
@@ -3,27 +3,26 @@ package com.gu.mediaservice.lib.elasticsearch
 import play.api.libs.json.Json
 
 object IndexSettings {
-  // TODO rename `english_s_stemmer` as its an analyzer not a stemmer - would require a reindex.
-  val guAnalyzer = "english_s_stemmer"
+  val englishAnalyzerName = "english_analyzer"
 
-  val englishSStemmer = Json.obj(
+  val englishAnalyzer = Json.obj(
     "type" -> "custom",
     "tokenizer" -> "standard",
     "filter" -> Json.arr(
-      "lowercase",
-      "asciifolding",
       "english_possessive_stemmer",
-      "gu_stopwords",
-      "s_stemmer"
+      "lowercase",
+      "english_stop",
```

Comment:
Having `lowercase` and `english_stop` in this order is important, as they are filtered and then sent to the stemmer.